
<launch>
	<!-- The motion Simulator -->
	<node name="CyberMotion" pkg="tk_CyberMotion" type="CyberMotion">
	
<!--
%machine limits of joint acc+vel
dqmax  = [    1.20427718387609	0.994837673636768	1.20427718387609	1.32645023151569	1.32645023151569	2.09439510239320	0.340000000000000 ];
ddqmax = [    1.71042266695444	1.22173047639603	2.23402144255274	0.575958653158129	1.65806278939461	1.34390352403563	2.50000000000000 ];
-->
		
		<param name="CyberMotionxPCBridge/txPCTargetHostName" value="localhost" />
		
		<param name="CyberMotionxPCBridge/tLowerPositionLimits" value="[-.inf,-2.23402144255274,-0.785398163397448,-3.14159265358979,-1.01229096615671,-3.14159265358979,0.186000000000000]" />
		<param name="CyberMotionxPCBridge/tUpperPositionLimits" value="[.inf,-0.837758040957278,1.60570291183478,3.14159265358979,1.01229096615671,3.14159265358979,1.54570000000000]" />
		
		<param name="CyberMotionxPCBridge/tLowerVelocityLimits" value="[-1.20427718387609,-0.994837673636768,-1.20427718387609,-1.32645023151569,-1.32645023151569,-2.09439510239320,-0.340000000000000]" />
		<param name="CyberMotionxPCBridge/tUpperVelocityLimits" value="[1.20427718387609,0.994837673636768,1.20427718387609,1.32645023151569,1.32645023151569,2.09439510239320,0.340000000000000]" />
		
		<param name="CyberMotionxPCBridge/tLowerAccelerationLimits" value="[-1.71042266695444,-1.22173047639603,-2.23402144255274,-0.575958653158129,-1.65806278939461,-1.34390352403563,-2.50000000000000]" />
		<param name="CyberMotionxPCBridge/tUpperAccelerationLimits" value="[1.71042266695444,1.22173047639603,2.23402144255274,0.575958653158129,1.65806278939461,1.34390352403563,2.50000000000000]" />
		
	</node>
</launch>
